wordpress-mobile/WordPress-iOS
# iOS 11: Improve search bar handling for Posts and Pages

Refs #7819.

This PR improves the content and scroll insets handling for the search bar in the Posts list and Pages list on iOS 11.

There are still a few odd things on iOS 11 – in particular, in landscape the search bar sits right at the top of the screen when search is active. I can't see any way around this though, as the system 'owns' the search bar at that point (it's in its own window). Ultimately I think the solution will be to move to the new `navigationItem.searchController` model, but that'll require some extra work.

![search-fix](https://user-images.githubusercontent.com/4780/31726568-3cf600a6-b41f-11e7-9d99-ec0d47070349.png)

To test:

* On iOS 10 and 11, with a range of devices, ensure that the content and scroll insets at the top of the Posts and Pages lists are correct when activating and deactivating search.

Needs review: @SergioEstevao
